server/core/src/notify_service_manager.cpp

```diff
@@ -0,0 +1,99 @@
+#include "irods/notify_service_manager.hpp"
+
+#include "irods/irods_logger.hpp"
+
+#include <cstddef>
+
+#ifdef IRODS_USE_LIBSYSTEMD
+
+#include <systemd/sd-daemon.h>
+
+namespace irods
+{
+    void do_notify_service_manager(const char* _msg,
+                                   [[maybe_unused]] const std::size_t _msg_length,
+                                   [[maybe_unused]] const char* _socket_path)
+    {
+        irods::experimental::log::server::debug("Notifying service manager: [{}].", _msg);
```

Comment:
Thanks for adding the periods. You can remove the trailing period on messages that end with:
```cpp
': [{}]"`
```

Reply:
Done